backendGameFetch.py

```diff
@@ -43,6 +41,16 @@
 }
 
 API_KEY='4ca42d11f7def3c9d0eea805aa48413a2c5ec7e6'
+GAMES_TO_FETCH = 10
+
+def create_conn(path):
+    connection = None
+    try:
+        connection = sqlite3.connect(path, check_same_thread=False)
+        print("DB connection successful.")
+    except Error as e:
+        print(f"An error '{e}' occurred during db connection")
```

Comment:
sometimes for security, the users are not allowed to know the reason why the database failed, but I think it is fine in this project.